CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.2.7 (2020-04-23)
+
+- Fixed the tabset component to set the active tab when `active` is set on the tab component. [#73](https://github.com/blackbaud/skyux-tabs/pull/73)
```

Comment:
The PR title makes it sound like the fix was on the tab component rather than the tabset component. But looking at the changes, it looks like we updated both. Would it be accurate to change this to something like `Fixed the tab and tabset components to set the active tab on initialization.`?

Reply:
Done.